doc/GettingStarted.md

```diff
@@ -122,6 +125,64 @@ In addition, Vitess requires the software and libraries listed below.
     ``` sh
     $ sudo apt-get install openjdk-7-jre
     ```
+    
+#### OS X
+
+1.  [Install Homebrew](http://brew.sh/).
+
+2.  On OS X, MySQL 5.6 has to be used, MariaDB doesn't work for some reason yet. It should be installed from Homebrew
+    (install steps are below).
+   
+3.  ZooKeeper is used as lock service.
+
+4.  Run the following commands:
+
+    ``` sh
+    brew install go automake libtool memcached python mercurial git bison curl wget homebrew/versions/mysql56
```

Comment:
To get this step to work, I had to first run `sudo chown -R $(whoami):admin /usr/local` as recommended here:

https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/El_Capitan_and_Homebrew.md

I guess you had already done that since you've used Homebrew before?